recipes/xilinx-runtime/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "xilinx-runtime" %}
+{% set version = "2.13.466" %}
+{% set quarter = "202210" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Xilinx/XRT/archive/refs/tags/{{ quarter }}.{{ version }}.tar.gz
+  sha256: d73d8366f6f6d66bfdcb67e72135f559a4d95644c5416c27dea79c235953fd20
+  patches:
+    - conda.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  ignore_run_exports_from:
+    - libdrm
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - pkg-config
+    - git
+
+  host:
+    - python
+    - pybind11
+    - boost
+    - boost-cpp
+    - protobuf
+    - libprotobuf
+    - rapidjson
+    - libuuid
+    - libdrm
+    - clhpp
+    - libcurl
+    - openssl
+    - ncurses
+
+  run:
+    - python
```

Ok, `boost-cpp` is a special package and has no run_exports as in most cases, only the headers are used. As it has a `pin_run_as_build` though, adding it as a plain dependency here is sufficient. If you also need Boost.Python, you will also need `boost` listed here.
```suggestion
    - python
    - boost
    - boost-cpp
```